torch/quantization/fake_quantize.py

```diff
@@ -111,7 +111,8 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
         for name in local_state:
             key = prefix + name
             if key in state_dict:
-                setattr(self, name, state_dict.pop(key))
+                val = state_dict[key]
```

Comment:
Can we remove this function?

Reply:
Removing this function throws an error which says that the  the size of the loaded tensor does not match the original size  i.e., These buffers start out with numel 0 and become numel 1 once they have their first forward pass.

Reply:
yeah, we can't, I have tried this before as well. Could you add a comment here? @supriyar